.travis.yml

```
@@ -24,6 +24,15 @@ matrix:
   - ghc: 8.6.5
     env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml" DEPLOY=yes
 
+  - language: nix
```

Maybe it doesn't use my `cachix` cache because by default Travi CI is using `nix-env` and `nix-build` version `2.0.4` but on my machine it's `2.3`
```suggestion
  - language: nix
    nix: 2.3
```